homepage/models.py

```diff
@@ -292,3 +292,42 @@ def landing_page_feed(cls):
     @staticmethod
     def user_already_posted_review(user_id, course_id):
         return True if Review.objects.filter(user=user_id, course=course_id) else False
+
+    def add_one_like(self):
+        self.likes_num = self.likes_num + 1

```

Comment:
I think add_like and remove_like sound nicer - but this is just a suggestion

Reply:
@VeraTs changed